### DISCUSSION TITLE
Is 20 tickets for being stuck on 20 kale from last season some kind of joke?

### DISCUSSION THREAD

**Discussion by NilzGone** [sunflower-land/sunflower-land#2523] [General]
1 day of time lost, and there's over 150 ticket difference between the people stuck on the kale quest and those who got to start the new season.

How does it equate to 20 tickets?

How didn't this come up at all from people with beta passes? Was their plan to screw some people over?

**Comment by ShiningMinions** [comment]
> How didn't this come up at all from people with beta passes? Was their plan to screw some people over?

Beta passers didn't test Dawn Breaker

**Reply by NilzGone** [reply]
> > How didn't this come up at all from people with beta passes? Was their plan to screw some people over?
> 
> Beta passers didn't test Dawn Breaker

The Live Stream has been delayed to fix some bugs related to Dawn Breaker season.

Players have discovered a range of issues that did not appear during **Beta testing.** The team is addressing these at the moment:

What was tested then?

**Reply by ShiningMinions** [reply]
> > > How didn't this come up at all from people with beta passes? Was their plan to screw some people over?
> > 
> > 
> > Beta passers didn't test Dawn Breaker
> 
> The Live Stream has been delayed to fix some bugs related to Dawn Breaker season.
> 
> Players have discovered a range of issues that did not appear during **Beta testing.** The team is addressing these at the moment:
> 
> What was tested then?

Not sure - just wanted to clear the air regarding beta pass holders.
